scripts/reindex.py

```diff
@@ -107,9 +101,9 @@ def main(argv: List[str]):
         logger.info('Queuing notifications for reindexing ...')
         for catalog in args.catalogs:
             if args.partition_prefix_length:
-                azul.remote_reindex(catalog, args.prefix, args.partition_prefix_length)
+                azul.remote_reindex(catalog, args.partition_prefix_length)
             else:
-                azul.reindex(catalog, args.prefix)
+                azul.reindex(catalog, '')
```

Comment:
This is the only call site so it doesn't make sense to fix this here but retain the argument. Why did you remove the args.reindex option?

Reply:
I think you are still reviewing this, but did you mean `args.prefix`?

Reply:
Uhm, yes. Good catch.

Reply:
I removed the `args.prefix`option because it seemed like it would not be needed if we hardcode the prefix to the source. By allowing a prefix to be provided to `azulclient.reindex` the reindexed bundles would be those under `{source.name.prefix} + {args.prefix}`. At the time I was thinking that if we are required to redeploy to update the the source prefix, why not just make this the only way to specify the prefix.

Maybe it's better to retain the args.prefix, make the default value of it `''`, and update the description to mention that its used to narrow down the source prefix, rather than to reindex an entirely different prefix.

> This is the only call site so it doesn't make sense to fix this here but retain the argument.

I would think that we want the azulclient to be robust enough to handle a prefix during reindex, so I opted to retain the prefix argument in that method vs removing it, even though its only used in this one call site.

I think there is a problem that we overload the term `prefix` in the repository plugins to mean `source.name.prefix` or `partition prefix`. By having distinct optional arguments for prefix and partition prefix within the methods of the repository plugin we can make make the client more flexible, and is probably something worth looking into in the long run.